guides/common/modules/proc_migrating-project-to-el-9-by-using-cloning.adoc

```diff
@@ -0,0 +1,20 @@
+[id="migrating-{project-context}-to-el-9-by-using-cloning_{context}"]
+= Migrating {Project} to {EL-abbr}{nbsp}9 by using cloning
+
+You can clone your existing {ProjectServer} from {EL}{nbsp}8 to a freshly installed {EL}{nbsp}9 system.
+Create a backup of the existing {ProjectServer}, which you then clone on the new {EL}{nbsp}9 system.
+[NOTE]
+====
+You cannot use cloning for {SmartProxyServer} backups.
+====
+
+.Procedure
+. Perform a full backup of your {ProjectServer}.
+This is the source {EL}{nbsp}8 server that you are migrating.
+For more information, see {AdministeringDocURL}Performing_a_Full_Backup_admin[Performing a full backup of {ProjectServer}] in _{AdministeringDocTitle}_.
+. Deploy a system with {EL}{nbsp}9 and the same configuration as the source server.
+This is the target server.
+. Clone the server.
+Clone configures hostname for the target server.
+
```

Comment:
```suggestion
```
This is just so that the link on line 20 appears as part of the whole step (to be consistent with the link on line 14).